File: Lecture 3/practice/SumOfDigits.java
import java.util.Scanner;

public class SumOfDigits{

    public static int sumDigits(int num){
        if(num == 0){
            return 0;
        }else{
            return num % 10 + sumDigits(num / 10);
        }
    }

    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        System.out.print("Number: ");
        sc.close();
            int num = sc.nextInt();
        System.out.print("Sum digits of " + num + " is " + sumDigits(num));
    }
}

File: Lecture 3/practice/SumOfDigits.py
#1. Tổng các chữ số của 1 số

def sumOfDigits(num):
    if num == 0:
        return 0
    else:
        return num % 10 + sumOfDigits(num // 10)

num = int(input("Number:"))
print("Sum of digits of", num, "is", sumOfDigits(num))